lib/uuidtools.rb

```diff
@@ -585,6 +585,8 @@ def self.ifconfig(all=nil)
 
       all_switch = all == nil ? "" : "-a"
       return `#{ifconfig_path} #{all_switch}` if not ifconfig_path == nil
+    rescue
+      ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.